src/EnergyPlus/WaterManager.cc

```diff
@@ -1000,7 +1000,12 @@ namespace WaterManager {
 
         if (RainFall.ModeID == RainSchedDesign) {
             schedRate = GetCurrentScheduleValue(RainFall.RainSchedID); // m/hr
-            ScaleFactor = RainFall.DesignAnnualRain / RainFall.NomAnnualRain;
+            if (RainFall.NomAnnualRain > 0.0){
+                ScaleFactor = RainFall.DesignAnnualRain / RainFall.NomAnnualRain;
+            }
+            else {
```

Comment:
If you could plop the `else {` up right after the `}` that would be great.  This change makes great sense.